repositories/trigger_message_repository.py

```
@@ -81,11 +81,11 @@ async def find(self, guild_id: str, trigger: str, channel_id: str) -> TriggerMes
             AND (tc.channel_id = %s)
         """
         params = (guild_id, trigger, channel_id)
-        result = await execute_query(query, params)
-        result = result[0] if result and result[0] else None
-        if not result:
+        rows_result: list[tuple[Any, ...]] | None = await execute_query(query, params)
```

Flake8 (7.3.0)

[error] undefined name 'Any'

(F821)